store/driver/tikv_driver.go

```diff
@@ -331,3 +334,12 @@ func (s *tikvStore) CurrentVersion(txnScope string) (kv.Version, error) {
 func (s *tikvStore) ShowStatus(ctx context.Context, key string) (interface{}, error) {
 	return nil, kv.ErrNotImplemented
 }
+
+func (s *tikvStore) GetLockWaits() []*deadlockPB.WaitForEntry {
+	stores := s.GetRegionCache().GetStoresByType(tikvrpc.TiKV)
+	resp, err := s.GetTiKVClient().SendRequest(context.TODO(), stores[0].Addr, tikvrpc.NewRequest(tikvrpc.CmdLockWaitInfo, &kvrpcpb.GetLockWaitInfoRequest{}), time.Second*30)
```

Comment:
Does anyone has any better idea of accessing the `wait_manager`? eg. An API to find which TiKV is `wait_manager` on?

Reply:
😕 I think you just collect from all TiKV nodes. The lock waiting information is in the wait_manager, which exists on all nodes. It's different from the deadlock detector, which has only one leader.